src/featureforest/utils/downloader.py

```diff
@@ -0,0 +1,45 @@
+from pathlib import Path
+import pooch
+
+
+MODELS_CACHE_DIR = Path.home().joinpath(".featureforest").joinpath("models")
```

Comment:
I like the '/' operator in pathlib, which is the same as 'joinpath'
So this like could be `Path.home() / ".featureforest" / "models"`
But no strong preference, if you think 'joinpath' is more readable, then just leave it as it is 😄

Reply:
Mmm, I'm not a fan of "/" 😁 It's a dividing operator!
Although, instead of using two consecutive `joinpath`, I used one with two params.